components/components.go

```diff
@@ -0,0 +1,107 @@
+package components
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-github/github"
+)
+
+// RepositoryToComponent is a list of all versioned deis repositories and their components.
+var RepositoryToComponent = map[string][]string{
+	"builder":          {"builder"},
+	"controller":       {"controller"},
+	"dockerbuilder":    {"dockerbuilder"},
+	"fluentd":          {"fluentd"},
+	"monitor":          {"influxdb", "grafana", "telegraf"},
+	"logger":           {"logger"},
+	"minio":            {"minio"},
+	"nsq":              {"nsqd"},
+	"postgres":         {"database"},
+	"redis":            {"loggerRedis"},
+	"registry":         {"registry"},
+	"router":           {"router"},
+	"slugbuilder":      {"slugbuilder"},
+	"slugrunner":       {"slugrunner"},
+	"workflow-manager": {"workflowManager"},
+}
```

Comment:
I'd be concerned that somebody would leave a repository out. Since most deisrel users will be building from source anyway, I'd think that the source would be more likely up to date than the file the user creates.